libsolidity/codegen/ir/IRGeneratorForStatements.cpp

```diff
@@ -302,6 +306,24 @@ void IRGeneratorForStatements::endVisit(FunctionCall const& _functionCall)
 			")\n";
 		break;
 	}
+	case FunctionType::Kind::Assert:
+	case FunctionType::Kind::Require:
+	{
+		solAssert(arguments.size() > 0, "Expected at least one parameter for require/assert");
+		solAssert(arguments.size() <= 2, "Expected no more than two parameters for require/assert");
+
+		string requireOrAssertFunction = m_utils.requireOrAssertFunction(
+			functionType->kind() == FunctionType::Kind::Assert,
+			arguments.size() > 1 ? arguments[1]->annotation().type : nullptr
+		);
+
+		m_code << move(requireOrAssertFunction) << "(" << m_context.variable(*arguments[0]);
+		if (arguments.size() > 1)
+			m_code << ", " << m_context.variable(*arguments[1]);
```

Comment:
I think this will generate invalid code if a string literal is used, but it is deactivated anyway, so it's fine.